### ISSUE TITLE
Hitting escape when editing a comment should confirm cancellation.

### ISSUE THREAD

**Issue by rwspeight-orange** [microsoft/vscode-pull-request-github#1455]
Hitting escape while editing a comment terminates edit mode and abandons unsaved work.  Confirming the cancellation with the user would help prevent lost work and user frustration.

**Comment by wcerfgba** [comment]
Yes please, this is easily the most frustrating problem I have with the extension at present. I can't tell you how many times I've accidentally knocked Esc and lost a PR comment with no way to recover it :sob:

**Comment by RikkiGibson** [comment]
It looks like this issue is still occurring. For comparison, if you hit esc while drafting a PR comment on the GitHub website, nothing happens.
